cmd/envtool/main.go

```diff
@@ -400,7 +223,7 @@ func main() {
 
 	logger := setupLogger(*debugLevel)
 
-	ctx, cancel := context.WithCancel(context.Background())
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
Would you mind to add add this `time.Minute` to the top of the file?

Reply:
Why? We use in a single place and its value is not configurable